pdns/iputils.hh

```diff
@@ -265,6 +265,7 @@ union ComboAddress {
   {
     char host[1024];
     int retval = 0;
+    // NOLINTNEXTLINE(cppcoreguidelines-pro-type-union-access)
```

Comment:
Could you add a comment, and also an issue if you think this is something we will change/fix in the future.

I added this bit of documentation for when it comes to silencing lint warnings: https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md#clang-tidy-and-ci

Reply:
Awesome, thank you!